inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/metrics/audit/AuditUtils.java

```diff
@@ -59,12 +57,32 @@ public static void initAudit() {
     }
 
     /**
-     * Add audit data
+     * Add input audit data
+     *
+     * @param event    event to be counted
      */
-    public static void add(int auditID, Event event) {
-        if (!CommonConfigHolder.getInstance().isEnableAudit() || event == null) {
+    public static void addInputSuccess(Event event) {
+        if (event == null || !CommonConfigHolder.getInstance().isEnableAudit()) {
             return;
         }
+        addAuditData(event,
+                AuditOperator.getInstance().buildSuccessfulAuditId(AuditIdEnum.DATA_PROXY_INPUT));
```

Comment:
It is recommended to calculate in advance and not to do it every time it is called